libraries/hermes/src/utils/ProtoGenerator.ts

```diff
@@ -83,6 +94,15 @@ export class ProtoGenerator {
     return `rpc ${newFunction}(${this.protoTemplate.request}) returns (${this.protoTemplate.response});\n`;
   }
 
+  private createProtoFunctionForProxy(path: ConduitProxy, protoFunctions: string) {
```

Comment:
you have created the generation function but you haven't modified the protoTemplate in the router or the admin router